spec/wordlist_dir_spec.rb

```diff
@@ -0,0 +1,135 @@
+require 'spec_helper'
+require 'ronin/wordlists/wordlist_dir'
+require 'ronin/wordlists/exceptions'
+
+describe Ronin::Wordlists::WordlistDir do
+  subject { described_class.new(path) }
+
+  let(:path) { File.expand_path(File.join(__dir__, '..', 'spec', 'fixtures', 'wordlists')) }
```

Comment:
Add a `let(:fixtures_dir)` for `spec/fixtures`. You can copy this from the other `*_spec.rb` files in `main`. I just added it to DRY things up.